src/data/valid/Biosample-exhaustive-issue-796-bye-yq-for-7-4-10.yaml

```diff
@@ -380,9 +379,9 @@ technical_reps: "2"
 analysis_type:
   - metabolomics
   - metagenomics
+
 sample_link:
-  - IGSN:DSJ0284
-  - any:curie
+  - 'parent: example_samp_45'
```

Comment:
parent is source_mat_id ... 1:1 parent to child. 
sample link is other sample types or horizontal relationships.. maybe make this one sibling: